ha-example/README.md

```diff
@@ -0,0 +1,160 @@
+# Flink cluster HA configuration
+
+In the deployment of Flink cluster, it consists of operators, job managers, and task managers.
+By default, the deployment is not a high availability (HA) cluster, we need to configure it to achieve it.
+
+## Flink operator
+
+The Flink operator manages and operates Flink Deployments. It supports high availability through leader election and standby operator instances.
+To enable leader election we need to add the following two mandatory operator configuration parameters.
+
+```
+kubernetes.operator.leader-election.enabled: true
+kubernetes.operator.leader-election.lease-name: flink-operator-lease
+```
+The lease name must be unique in the current lease namespace.
```

Comment:
What is the "lease namespace". Is it the ns where the operator is deployed?

Reply:
Yes, it is. Let me update it. Thanks.